js/adapt-articleView.js

```diff
@@ -1,508 +1,497 @@
+          _index: i,
+          _includeNumber: i !== 0,
+          _title: blocks[i].get('title')
+        });
+      }
 
-        },
+      this.model.set({
+        _currentBlock: 0,
+        _totalBlocks: totalBlocks,
+        _itemButtons: itemButtons
+      });
+    },
 
-        _onBlockSliderReady: function() {
-            this._blockSliderHideOthers();
-            _.delay(_.bind(function(){
-                this._blockSliderConfigureControls(false);
-                this._onBlockSliderResize();
-            },this),250);
-            this.$(".component").on("resize", this._blockSliderResizeHeight);
-        },
+    _blockSliderConfigureControls: function() {
 
-        _onBlockSliderClick: function(event) {
-            event.preventDefault();
+      var duration = this.model.get("_articleBlockSlider")._slideAnimationDuration || 200;
 
-            var id = $(event.currentTarget).attr("data-block-slider");
+      if (this._disableAnimationOnce) animate = false;
+      var _currentBlock = this.model.get("_currentBlock");
+      var _totalBlocks = this.model.get("_totalBlocks");
 
-            switch(id) {
-            case "left":
-                this._blockSliderMoveLeft();
-                break;
-            case "index":
-                var index = parseInt($(event.currentTarget).attr("data-block-slider-index"));
-                this._blockSliderMoveIndex(index);
-                break;
-            case "right":
-                this._blockSliderMoveRight();
-                break;
-            }
+      var $left = this.$el.find("[data-block-slider='left']");
+      var $right = this.$el.find("[data-block-slider='right']");
 
-        },
+      if (_currentBlock === 0) {
+        $left.a11y_cntrl_enabled(false);
+        $right.a11y_cntrl_enabled(true);
+      } else if (_currentBlock == _totalBlocks - 1 ) {
+        $left.a11y_cntrl_enabled(true);
+        $right.a11y_cntrl_enabled(false);
+      } else {
+        $left.a11y_cntrl_enabled(true);
+        $right.a11y_cntrl_enabled(true);
+      }
 
-        _blockSliderMoveLeft: function() {
-            if (this.model.get("_currentBlock") === 0) return;
+      var $indexes = this.$el.find("[data-block-slider='index']");
+      $indexes.a11y_cntrl_enabled(true).removeClass("is-selected");
+      $indexes.eq(_currentBlock).a11y_cntrl_enabled(false).addClass("is-selected is-visited");
 
-            var index = this.model.get("_currentBlock");
-            this._blockSliderMoveIndex(--index);
-        },
+      var $blocks = this.$el.find(".block");
+      if (!$blocks.length) return;
 
-        _blockSliderMoveIndex: function(index, animate) {
-            if (this.model.get("_currentBlock") != index) {
+      $blocks.a11y_on(false).eq(_currentBlock).a11y_on(true);
 
-                this.model.set("_currentBlock", index);
+      if (Adapt.accessibility.isActive()) {// prevents https://github.com/cgkineo/adapt-articleBlockSlider/issues/28
```

Comment:
`Adapt.accessibility.isActive()` [will always return false in Adapt v5](https://github.com/adaptlearning/adapt_framework/blob/7c7019c8d007a788354465ef33c115d02210ab4b/src/core/js/a11y.js#L153-L156), so this whole `if` block will never execute - might as well get rid of it.